### ISSUE TITLE
`release-as` in monorepo

### ISSUE THREAD

**Issue by nugaon** [google-github-actions/release-please-action#590]
### TL;DR

It is more like a question because I haven't found the solution in the documentation, but how should I use the `release-as: 0.1.0` commit message if I have multiple projects in the repo?

I tried to prefix the commit message with the `blossom-lib` monorepo tag:
`git commit --allow-empty -m "chore: release blossom-lib 0.1.0" -m "Release-As: 0.1.0"`

it changed the [release PR of another project](https://github.com/nugaon/blossom/pull/1) instead

Will it release with the correct version if I just manually edit the version in the PR title as well as its auto-generated changes in the project config files?

**Comment by chingor13** [comment]
An empty commit applies to all components of the monorepo. Otherwise, the commit will apply to components whose code paths have been touched. So if you want to use a `Release-As` commit on a single component, you'll have to touch a file in that component's path.
